packages/app/src/systems/Account/components/FuelAddress/FuelAddress.tsx

```diff
@@ -28,20 +28,20 @@ export const FuelAddress = ({ address, css }: AddressProps) => {
           <Text css={css}>{shortAddress(fuelAddress)}</Text>
         </Tooltip>
       </Copyable>
-    </Flex>
+    </Box.Flex>
   );
 };
 
 const styles = {
-  root: {
+  root: cssObj({
     '.address_tooltip': cssObj({
-      fontSize: '$xs',
+      fontSize: '$sm',
       lineHeight: '$4',
       maxWidth: 125,
```

Comment:
In some cases we don't want to use variables, to avoid the screen breaking. We should be really constrain in the cases where we are setting padding, margins and other stuff like this but for height, width it's ok to use other values as this are Application specific.